fontbe/src/glyphs.rs

```diff
@@ -731,22 +742,62 @@ fn affine_for(component: &Component) -> Affine {
     ])
 }
 
-fn bbox2rect(bbox: Bbox) -> Rect {
-    Rect {
-        x0: bbox.x_min.into(),
-        y0: bbox.y_min.into(),
-        x1: bbox.x_max.into(),
-        y1: bbox.y_max.into(),
-    }
-}
-
 #[derive(Debug)]
 struct GlyfLocaWork {}
 
 pub fn create_glyf_loca_work() -> Box<BeWork> {
     Box::new(GlyfLocaWork {})
 }
 
+/// See <https://github.com/fonttools/fonttools/blob/42c1a52c5facd0edbc9c685787b084af44f6f607/Lib/fontTools/ttLib/tables/_g_l_y_f.py#L1244>
+fn bbox_of_composite(
+    glyph_order: &GlyphOrder,
+    glyphs: &HashMap<&GlyphName, Arc<Glyph>>,
+    composite: &CompositeGlyph,
+    affine: Affine,
+) -> Result<Option<Rect>, Error> {
+    // For simple scale+translate transforms, which seem to be common, we could just transform a bbox
+    // Let's wait to see if that pops out in a profile and do the simple solution for now
+    // Because transforms can skew/rotate the control box computed for the simple glyph isn't always reusable
+
+    let mut bbox: Option<Rect> = None;
+    for component in composite.components() {
+        // The transform we get here has changed because it got turned into F2Dot14 and i16 parts
+        // We could go get the "real" transform from IR but ... this seems to match fontmake so far
```

Comment:
the 'real' transform from source or IR doesn't actually matter any more, it's the one that's stored in the glyf table that matters for bbox computation, that's what fonttools TTGlyph.recalcBounds() does (and ufo2ft uses the same method on the compiled TTGlyphs when building the glyf table).